sentences/fr/valve_HassTurnOff.yaml

```diff
@@ -0,0 +1,11 @@
+language: fr
+intents:
+  HassTurnOff:
+    data:
+      - sentences:
+          - "ferme [<le>]{name}"
```

Comment:
Same comment as for HassSetPosition : Verbs should be expansion rules and support infinitive forms